### PR TITLE
[Code health] Lint checks & CI fix

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
@gino-m  PTAL?

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -74,6 +74,10 @@ steps:
         ./gradlew -PdisablePreDex checkCode 2> check-logs.txt || echo "fail" > build-status.txt
         cat check-logs.txt
 
+        if [[ "${_PUSH_TO_MASTER}" ]]; then
```
Comment (gino-m) [inline cloudbuild.yaml:77]: Could you please add a comment explaining when _PUSH_TO_MASTER is true? Is it run before GitHub merges an approved PR with master? Will updates block merge?

**Review by gino-m — CHANGES_REQUESTED** [review]

File: cloudbuild.yaml (hunk 1 above)
Comment (shobhitagarwal1612) [reply cloudbuild.yaml:81]: Added notes. No, it won't block merges.

**Comment by shobhitagarwal1612** [comment]
Needs a re-approval

**Review by gino-m — APPROVED** [review]
Restamp!

**Pull request merged**